### DISCUSSION TITLE
PimCore admin backend doesn't work anymore

### DISCUSSION THREAD

**Discussion by scorpion97429** [pimcore/pimcore#10133] [Q&A / Support]
Hello,

After succesfull install i was testing Pimcore. Then after a moment the admin backend stopped working.
I'm using a virtual machine and even if i use a snapshot that was working just after the install it don't work anymore.

Front-end work correctly.

I just got that :

![pimcore](https://user-images.githubusercontent.com/7247076/130595744-42ba36b6-53ba-4dca-9e70-6a84d6d35092.jpg)

**Comment by dvesh3** [comment]
Seems like file permission issue https://pimcore.com/docs/pimcore/current/Development_Documentation/Installation_and_Upgrade/System_Setup_and_Hosting/File_Permissions.html . Please check logs under `/var/log` folder for more info.
